Gamedata/Bluedog_DB/Parts/Atlas/bluedog_Atlas_SustainerAdapterTank.cfg

```diff
@@ -34,7 +34,8 @@ MODEL
 	bulkheadProfiles = size1, size1p5
 	
 	tags = fueltank lfo liquid oxidizer propellant rocket ?atlas bossart missile 1.875 1875 1.25 125 adapter ?slv3x
-	
+	techtag = atlasD
```

Comment:
Suggest putting into atlasA

Reply:
Technically Atlas A can be built without the adapter tank but the booster skirt will clip in and you have to use the offset tool to shift it down which seems a bit gross. Since the LR105 sustainer is in another node you are still gated from building a half stage atlas and you get a cleaner build.

Reply:
Done.